packages/webpack-scaffold/index.ts

```diff
@@ -0,0 +1,121 @@
+import jscodeshift from "jscodeshift";
+
+interface IBaseObj {
+	type: string;
+	name: string;
+	message: string;
+}
+
+interface IList extends IBaseObj {
+	choices?: string[];
+}
+
+interface IInput extends IBaseObj {
```

Comment:
better convention here. `Inq_Input` etc maybe

Reply:
What "Inq" stands for?

Reply:
I think Inquirer?